macaw-aarch32-symbolic/src/Data/Macaw/AArch32/Symbolic/ABI.hs

```diff
@@ -0,0 +1,249 @@
+{-|
```

Comment:
@RyanGlScott There's a lot of code here that's basically equivalent to the code for the x86_64 SysV ABI, particularly the part about writing spilled arguments to the stack while maintaining proper alignment. Do you have thoughts on the best way to share between the two (or if we should share it)? I could lift `writeSpilledArgs` to `Data.Macaw.Symbolic.Stack` perhaps, with a note that it's only appropriate for use on certain ABIs, and that you'd probably want to use the specializations of it that we would provide in `Data.Macaw.{X86,AArch32}.Symbolic.ABI`?

Reply:
I think putting the shared code (with caveats that it is only valid for certain ABIs) in `macaw-symbolic` is fine.